media/js/src/timer.js

```diff
@@ -1,27 +1,40 @@
 let time = $('meta[name="time"]').attr('content');
```

Comment:
I think even these meta tags are considered DOM elements, I would move these two lines to the beginning of your `$(document).ready` function.

Reply:
True!